### PR TITLE
[strings] Fish pass translations

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

**Review by gdt — COMMENTED** [review]
I do not feel particularly competent to review this, but I have looked it over and I do not see anything that I think is incorrect.

I am very unclear on "English is always Title Case", so I decline to comment on that (L162) line.

**Comment by biodranik** [comment]
> I am very unclear on "English is always Title Case", so I decline to comment on that (L162) line.

Many years ago we decided that the feature type name looks better if every word is capitalized: `Title Case` compared to `Title case`.

**Review by Wivik — APPROVED** [review]
OK for French.

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -26339,7 +26339,45 @@
...
+    it = Scala per pesci
+    ja = フィッシュラダー
+    ko = 물고기 사다리
+    lt = Žuvų kopėčios
+    mr = मासे शिडी
+    nb = Fiskestige
+    nl = Vistrap
+    pl = Drabina dla ryb
+    pt = Escada de peixe
+    pt-BR = Escada para peixes
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:26369]: pt = Escada de peixes
pt-BR = Escada de peixes

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -26339,7 +26339,45 @@
...
+    ro = Scară de pește
+    ru = Рыбная лестница
```
Comment (pastk) [inline data/strings/types_strings.txt:26371]: Рыбоход
https://ru.wikipedia.org/wiki/%D0%A0%D1%8B%D0%B1%D0%BE%D0%BF%D1%80%D0%BE%D0%BF%D1%83%D1%81%D0%BA%D0%BD%D1%8B%D0%B5_%D1%81%D0%BE%D0%BE%D1%80%D1%83%D0%B6%D0%B5%D0%BD%D0%B8%D1%8F

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -26339,7 +26339,45 @@
...
+    sk = Rebrík pre ryby
+    sv = Fiskstege
+    sw = Ngazi ya samaki
+    th = บันไดปลา
+    tr = Balık merdiveni
+    uk = Риб'ячі сходи
```
Comment (pastk) [inline data/strings/types_strings.txt:26377]: Рибохід

**Review by jimcarst — APPROVED** [review]

File: data/strings/types_strings.txt [hunk 4]
```diff
@@ -26339,7 +26339,45 @@
...
+    hu = Hal létra
+    id = Tangga ikan
+    it = Scala per pesci
+    ja = フィッシュラダー
+    ko = 물고기 사다리
+    lt = Žuvų kopėčios
+    mr = मासे शिडी
+    nb = Fiskestige
+    nl = Vistrap
+    pl = Drabina dla ryb
```
Comment (RicoElectrico) [inline data/strings/types_strings.txt:26367]: it should be "Przepławka" according to PL Wikipedia.

File: android/res/values-be/strings.xml [hunk 5]
```diff
@@ -1167,6 +1167,7 @@
 	<string name="type.tourism.information.guidepost">Даведнік</string>
 	<string name="type.tourism.information.map">Карта</string>
 	<string name="type.tourism.information.office">Турыстычны офіс</string>
+	<string name="type.waterway.fish_pass">Рыбная лесвіца</string>
```
Comment (zubialevich) [inline android/res/values-be/strings.xml:1170]: Рыбаход

https://verbum.by/tsbm/rybachod

File: data/strings/types_strings.txt [hunk 6]
```diff
@@ -26339,7 +26339,45 @@
 
   [type.waterway.fish_pass]
     en = Fish Pass
+    ar = سلم السمك
+    be = Рыбная лесвіца
```
Comment (zubialevich) [inline data/strings/types_strings.txt:26343]: Рыбаход 

https://verbum.by/tsbm/rybachod

**Review by vng — APPROVED** [review]

**Pull request merged**